index.html

```diff
@@ -1,17 +0,0 @@
----
```

Comment:
I had created index.html and renamed it. I should have done git rm but did
not since I'm a sloppy coder and bad at Git. It shouldn't conflict it's
just my bad git. Sorry for the confusion.

On Tue, Nov 17, 2015 at 22:56 Mason Donahue <notifications@github.com>
wrote:

> In index.html
> <https://github.com/lucyparsons/lucyparsons.github.io/pull/19#discussion_r45159898>
> :
>
> > @@ -1,17 +0,0 @@
> > ----
>
> do we not need this page?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lucyparsons/lucyparsons.github.io/pull/19/files#r45159898>
> .
>